### ISSUE TITLE
Mouse 4 and 5 navigate both panes in Dual Pane View.

### ISSUE THREAD

**Issue by tgrelka** [files-community/Files#2910]
**Describe the bug**
When using dual pane mode, mouse buttons 4 and 5 (used for navigation, typically on the left side of the mouse) do not only navigate the current pane, but both at the same time.

**To Reproduce**
1. Open a tab in dual pane view.
2. Navigate using mouse buttons 4 and 5.
3. Watch both panes change path.

**Expected behavior**
I want only the currently focussed pane to navigate, since it can be quite annoying when you’re searching for a specific folder in a pane to use, e.g. to copy files to or from, and don’t want the other path to change in tandem. Generally, the paths in the panes are different from each other and navigating them both at the same time doesn’t make much sense.

**Desktop:**
 - OS Version: Windows 10 Build 19042.685
 - App version: v1.0.0.0

**Log file**
Don’t think it’s of any interest, but for good measure and a complete bug report:
```
2021-01-11 06:39:34.1239|INFO|.ctor|App launched|
2021-01-11 06:39:34.2869|INFO|.ctor|Drive added: C:\, Fixed|
```

**Comment by generalguy41** [comment]
To add to this in 1.1.0.0, it also navigates other open tabs (See the second tab in recording).

https://user-images.githubusercontent.com/8879244/106361746-5f441280-62ed-11eb-9c86-c840631028b2.mp4

**Comment by yaichenbaum** [comment]
@tgrelka This issue should be resolved with #3465.
